### PR TITLE
Updated README.md content

### PR DESCRIPTION
**Pull request by susanmdouglas-aws**
Signed-off-by: susanmdouglas <susandou@amazon.com>

### Description
Updated README.md; Proofread content and fixed grammar and typos where appropriate.  Refined build sections, adding links to installation information at product sites.  Separated Writing Guidelines section into bulleted lists.  Removed Testing section, pending further discussion with team.
 
### Issues Resolved
Adds links to installation documentation for Jekyll/Docker, reorganizes content into cohesive sections, improved grammar and voice.

### Check List
- [x ] Commits are signed per the DCO using `--signoff`

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

**Review by 3manuek — APPROVED** [review]

**Review by hsuamz — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -17,158 +17,147 @@
...
-If you contribute code, make sure that you are not violating 3rd party rights.
-Checking for intellectual property violations beforehand is expected by the community. We want
-Babelfish and our users to be free of 3rd party claims. Keeping the code clean benefits 
-the community as well our users.
-
-This project has adopted the [Amazon Open Source Code of Conduct](CODE_OF_CONDUCT.md). For more information see the [Code of Conduct FAQ](https://aws.github.io/code-of-conduct-faq), or contact [opensource-codeofconduct@amazon.com](mailto:opensource-codeofconduct@amazon.com) with any additional questions or comments.
+Babelfish would not be possible without the hard work and dedication of the people who have contributed to PostgreSQL. Everyone involved in the development of PostgreSQL has our gratitude.
 
-This website was forked from the BSD-licensed [djangoproject.com](https://github.com/django/djangoproject.com) and merged with just-the-docs theme.
+This website was forked from the BSD-licensed [djangoproject.com](https://github.com/django/djangoproject.com) and merged with the just-the-docs theme.
```
Comment (3manuek) [inline README.md:153]: Here it is the [just-the-docs](https://github.com/pmarsceill/just-the-docs) link. cc / @susanmdouglas-aws @gopinathpai

**Review by 3manuek — DISMISSED** [review]

File: README.md [hunk 2]
```diff
@@ -17,158 +17,162 @@
...
 ## Credits
 
-If you contribute code, make sure that you are not violating 3rd party rights.
-Checking for intellectual property violations beforehand is expected by the community. We want
-Babelfish and our users to be free of 3rd party claims. Keeping the code clean benefits 
-the community as well our users.
+If you contribute code, make sure that your code does not violate 3rd party rights; check for intellectual property conflicts before you submit a pull request that includes content from other projects. 
 
-This project has adopted the [Amazon Open Source Code of Conduct](CODE_OF_CONDUCT.md). For more information see the [Code of Conduct FAQ](https://aws.github.io/code-of-conduct-faq), or contact [opensource-codeofconduct@amazon.com](mailto:opensource-codeofconduct@amazon.com) with any additional questions or comments.
+This project has adopted the [Open Source Code of Conduct](CODE_OF_CONDUCT.md). For more information see the [Code of Conduct FAQ](https://aws.github.io/code-of-conduct-faq), or contact [opensource-codeofconduct@amazon.com](mailto:opensource-codeofconduct@amazon.com) with any additional questions or comments.
```
[the remark this answers is not in the document]
Comment (susanmdouglas-aws) [reply README.md]: I've actually refactored that in the most-recent check in, though I don't see it here... let me make sure my check in made it to Github...

File: README.md (hunk 1 above)
Comment (susanmdouglas-aws) [reply README.md]: Added....

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (susanmdouglas-aws) [reply README.md]: In the last iteration, reads:

This project has adopted the Open Source Code of Conduct; for more information see the [Code of Conduct](https://aws.github.io/code-of-conduct.html).

**Review by 3manuek — APPROVED** [review]

**Pull request merged**